.github/workflows/release-cli.yml

```diff
@@ -127,11 +127,6 @@ jobs:
           echo "$APK_KEY_PRIVATE" > /tmp/keys/apk.key
           chmod 0600 /tmp/keys/apk.key
 
-      - name: Install Gon
-        run: |
-          brew tap mitchellh/gon
-          brew install mitchellh/gon/gon
-
```

Comment:
`gon` tool work no more, `notarytool` should be already present in the CI (let's test it).